cmd/openshift-install/create.go

```diff
@@ -427,8 +427,8 @@ func logComplete(directory, consoleURL string) error {
 		return err
 	}
 	logrus.Info("Install complete!")
-	logrus.Infof("Run 'export KUBECONFIG=%s' to manage the cluster with 'oc', the OpenShift CLI.", kubeconfig)
-	logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s' succeeds (wait a few minutes).", pw)
```

Comment:
+1 to dropping this line.  Is the motivation "the web console will be up and running by the time the CVO claims success, so we don't need to poll this to find out"?

Reply:
Yes, this line was added before we checked the cluster initialization via clusterversion/clusteroperators available.  It's outdated, has to go : )

Reply:
answered somewhere, GH  has confused me.. lol but yes, that line is outdated.  That line was added before we properly waited for clusterversion/all clusteroperators available.